python/lsst/cp/pipe/cpCombine.py

```diff
@@ -341,6 +341,18 @@ def run(self, inputExpHandles, inputScales=None, inputDims=None):
         self.combineHeaders(inputExpHandles, combinedExp,
                             calibType=self.config.calibrationType, scales=expScales)
 
+        # Populate a visitInfo.  Set the exposure time and dark time
+        # to 0.0 or 1.0 as appropriate, and copy the instrument name
+        # from one of the inputs.
+        expTime = 1.0
+        if self.config.connections.outputData.lower() == 'bias':
+            expTime = 0.0
+        inputVisitInfo = inputExpHandles[0].get(component="visitInfo")
```

Comment:
Would be convenient if we didn't have to do this because combineHeaders returned some kind of combo VisitInfo as well as the modified header.

Reply:
I've moved this into the `combineHeaders` call, as that has the `VisitInfo` list already available, to avoid the unnecessary extra `get`.